signatures/LUOV/LUOV/impl/avx2-keccak.yaml

```diff
@@ -0,0 +1,4 @@
+name: avx2-keccak
```

Comment:
I strongly assume the keccak and chacha8 versions have different input and output. I would say they are different flavors then. At the very least, it's different parameter sets.